packages/backend/src/ee/database/migrations/20250516162927_add_ai_agent_integrations_table.ts

```diff
@@ -0,0 +1,27 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+    if (await knex.schema.hasTable('ai_agent_integration')) return;
```

Comment:
```suggestion
    if (await knex.schema.hasTable(AiAgentIntegrationTableName)) return;
```

would be cool to import these types of constants from `entities/aiAgent.ts`